### DISCUSSION TITLE
How to make new window be place only at workspace where it was opened?

### DISCUSSION THREAD

**Discussion by ghost** [i3/i3#5898] [Q&A] [answered]
Hello.

Please, assume 2 next next simple scenarios.

# 1

On workspace `a` some long-starting app was requested to open. User has been switched to the workspace `b` and requested at workspace `a` window appeared.

# 2

On workspace `a` I have opened some app. I have switched to workspace `b` and trying to open several new apps with the same type - but nothing happens. In fact, they were already opened at workspace `b`.

How to handle these scenarios in correct way?

Hope on your reply.

**Comment by okraits** [comment, marked as answer]
Scenario 1: https://i3wm.org/docs/userguide.html#assign_workspace explaines some options.

Scenario 2: I guess you will need some setting in the specific app to control wether a new instance will be opened in the same workspace as the existing instance or in the current workspace.
